src/main/scala/org/apache/spark/scheduler/cluster/armada/ArmadaEventWatcher.scala

```diff
@@ -0,0 +1,335 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.scheduler.cluster.armada
+
+import java.util.concurrent.ConcurrentHashMap
+
+import scala.util.control.NonFatal
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+import api.event.{
+  EventGrpc,
+  WatchRequest,
+  EventMessage,
+  JobSubmittedEvent,
+  JobQueuedEvent,
+  JobPendingEvent,
+  JobRunningEvent,
+  JobSucceededEvent,
+  JobFailedEvent,
+  JobCancelledEvent,
+  JobPreemptingEvent,
+  JobPreemptedEvent,
+  JobTerminatedEvent,
+  JobUnableToScheduleEvent
+}
+import io.grpc.ManagedChannel
+// import io.grpc.stub.StreamObserver  // no longer needed; switched to blocking API
+import org.apache.spark.internal.Logging
+
+/** Minimal Armada event watcher placeholder.
+  *
+  * This class is intended to be started by ArmadaClusterManagerBackend. For now, it starts a daemon
+  * thread and keeps it alive until stopped. The actual event handling will be implemented with the
+  * current Armada Scala client API.
+  */
```

Comment:
Just trying to verify my understanding, is this docstring still correct? 
>The actual event handling will be implemented with the current Armada Scala client API.

Does that mean we would need to implement event handling later on?